object-cache.php

```diff
@@ -1256,13 +1256,15 @@ public function build_client_parameters( $redis_server ) {
 			if ( isset( $_SERVER['CACHE_HOST'] ) ) {
 				$redis_server = [
 					// Don't use WP methods to sanitize the host due to plugin loading issues with other caching methods.
-					// @phpcs:ignore WordPressVIPMinimum.Functions.StripTags.StripTagsOneParameter,WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
-					'host' => strip_tags( $_SERVER['CACHE_HOST'] ),
-					'port' => ! empty( $_SERVER['CACHE_PORT'] ) ? intval( $_SERVER['CACHE_PORT'] ) : $port,
+					// @phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized,WordPress.Security.ValidatedSanitizedInput.InputNotValidated
+					'host' => ( function_exists( 'wp_strip_all_tags' ) ? wp_strip_all_tags( wp_unslash( $_SERVER['CACHE_HOST'] ) ) : ( function_exists( 'wp_unslash' ) ? wp_unslash( $_SERVER['CACHE_HOST'] ) : $_SERVER['CACHE_HOST'] ) ),
```

Comment:
Does object-cache eval before WordPress is fully bootstrapped? function_exists feels overzealous but maybe needed here.

Also all these nested ternary operators are not particularly readable. Maybe operate on variables before assigning to $redis_server? at first glance I wasn't clear if we were running both methods or falling back here...